src/Java.Interop.Dynamic/Tests/Java.Interop.Dynamic-Tests.csproj

```diff
@@ -36,9 +36,6 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
-    <Reference Include="nunit.framework">
-      <HintPath>..\..\..\lib\NUnit-2.6.3\bin\nunit.framework.dll</HintPath>
```

Comment:
BTW nunit.framework.dll *IS* still present in Mono 4.9, it's just that types like TestFixtureAttribute have the Obsolete attribute applied with IsError=true. In this case I guess what happens is that since the HintPath doesn't exist it'll instead use the assembly from the GAC, which results in the obsolete error.